README.rst

```diff
@@ -117,6 +117,9 @@ Edit your settings.py file:
     # https://cloud.google.com/docs/authentication/getting-started>
 
     FCM_DJANGO_SETTINGS = {
+         # an instance of firebase_admin.App to be used as default for all fcm-django requests
+         # default: None (the default Firebase app)
+        "DEFAULT_FIREBASE_APP": None,
```

Comment:
Would it perhaps be helpful to add an example of how the setting looks like when it is not set to `None`?

Reply:
Maybe in the README, because here's the code for the first setup, and I didn't want to complicate things for people who just want to configure fcm-django with the default app.

Reply:
Yes that makes sense

Reply:
I have updated the README with a new section. I don't know if my way of creating a Credentials object is standard, but I haven't found anything on Google, so I just inherited and customized the default Google one.